universal-login-react/src/ui/Onboarding/ReactCodeInput.tsx

```diff
@@ -345,30 +351,32 @@ export class ReactCodeInput extends Component<ReactCodeInputProps, any> {
       <div className={classNames(className, 'react-code-input')} style={styles.container}>
         {input.map((value: any, i: number) => {
           return (
-            <input
-              ref={(ref) => {
-                this.textInput[i] = ref!;
-              }}
-              id={`${this.uuid}-${i}`}
-              data-id={i}
-              autoFocus={autoFocus && i === 0}
-              value={value}
-              key={`input_${i}`}
-              type={type}
-              min={0}
-              max={9}
-              maxLength={input.length === i + 1 ? 1 : input.length}
-              style={styles.input}
-              autoComplete="off"
-              onFocus={(e) => e.target.select()}
-              onBlur={(e) => this.handleBlur(e)}
-              onChange={(e) => this.handleChange(e)}
-              onKeyDown={(e) => this.handleKeyDown(e)}
-              disabled={disabled}
-              data-valid={isValid}
-              pattern={pattern}
-              inputMode={inputMode}
-            />
+            <div className={`${wrapperClassName || 'react-code-wrapper'} ${value && 'it-has-value'}`} style={{display: 'inline-block'}} key={`input_${i}`}>
+              <input
+                ref={(ref) => {
+                  this.textInput[i] = ref!;
+                }}
+                id={`${this.uuid}-${i}`}
+                data-id={i}
+                autoFocus={autoFocus && i === 0}
+                value={value}
+                type={type}
+                min={0}
+                max={9}
+                maxLength={input.length === i + 1 ? 1 : input.length}
+                className={inputClassName}
+                style={inputClassName !== '' ? {} : styles.input}
```

Comment:
We can simplify it: `style={inputClassName || styles.input}`

Reply:
This case don't work, because style can't be undefined. I improved this line in this way:
```style={inputClassName ? styles.input : {}}```